<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>PropertyWidget_PathTextBase</class>
 <widget class="QFrame" name="PropertyWidget_PathTextBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>243</width>
    <height>143</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <property name="leftMargin">
    <number>3</number>
   </property>
   <property name="topMargin">
    <number>3</number>
   </property>
   <property name="rightMargin">
    <number>3</number>
   </property>
   <property name="bottomMargin">
    <number>3</number>
   </property>
   <property name="spacing">
    <number>3</number>
   </property>
   <item row="0" column="0">
    <widget class="QLabel" name="pathTextTypeLabel">
     <property name="text">
      <string>Type:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="pathTextType"/>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="startOffsetLabel">
     <property name="text">
      <string>Start Offset:</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="ScrSpinBox" name="startOffset"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="distFromCurveLabel">
     <property name="text">
      <string>Distance from Curve:</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="ScrSpinBox" name="distFromCurve"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="flippedPathText">
     <property name="text">
      <string>Flip Text</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QCheckBox" name="showCurveCheckBox">
     <property name="text">
      <string>Show Curve</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>ScrSpinBox</class>
   <extends>QDoubleSpinBox</extends>
   <header>ui/scrspinbox.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
